tsconfig.base.json

```diff
@@ -20,42 +17,35 @@
     "baseUrl": "packages",
     "downlevelIteration": true,
     "paths": {
```

Comment:
why were some of these entries removed from `paths`, not opposed, just curious

Reply:
I have not deleted any, just sorted them alphabetically, so  that the next time it's easier to discover if a package is missing. But because of this comment I realized I had some mistakes in package names and fixed them.